Signal/src/ViewControllers/AppSettings/BlockListViewController.m

```diff
@@ -78,18 +79,22 @@ - (void)updateTableContents
     NSArray<NSString *> *blockedPhoneNumbers =
         [helper.blockedPhoneNumbers sortedArrayUsingSelector:@selector(compare:)];
     for (NSString *phoneNumber in blockedPhoneNumbers) {
-        [blocklistSection addItem:[OWSTableItem itemWithCustomCellBlock:^{
-            ContactTableViewCell *cell = [ContactTableViewCell new];
-            SignalAccount *signalAccount = [helper signalAccountForRecipientId:phoneNumber];
-            if (signalAccount) {
-                [cell configureWithSignalAccount:signalAccount contactsManager:helper.contactsManager];
-            } else {
-                [cell configureWithRecipientId:phoneNumber contactsManager:helper.contactsManager];
-            }
-
-            return cell;
-        }
-                                      customRowHeight:[ContactTableViewCell rowHeight]
+        [blocklistSection addItem:[OWSTableItem
```

Comment:
Is your clang format feeling OK?

I never really know what to expect, so this could be correct... just wanted to double check.

Reply:
I have no idea how to determine that.  It's a mysterious beast.